### PR TITLE
DOC-352 Replace NGINX with Envoy

### PR DESCRIPTION
**Pull request by bmansheim**
Make references to web proxy generic

### CODE REVIEW

File: content/ri/using-redisinsight/troubleshooting.md [hunk 1]
```diff
@@ -27,10 +27,10 @@ You can install RedisInsight on operating systems that are not officially suppor
 
 We are happy to receive your feedback at redisinsight@redislabs.com.
 
-
 ## Using behind a reverse proxy
 
-RedisInsight should work fine behind a reverse proxy like Nginx for most use-cases.
+RedisInsight should work fine behind a reverse proxy like NGINX for most use-cases.
```
Comment (AlonMagrafta) [inline content/ri/using-redisinsight/troubleshooting.md:32]: I just don't know - NGINX or Envoy?

Comment (bmansheim) [reply content/ri/using-redisinsight/troubleshooting.md:32]: This is just giving an example of a proxy that you can setup RedisInsight behind. I think NGINX is fine.

**Review by AlonMagrafta — APPROVED** [review]

File: content/ri/using-redisinsight/troubleshooting.md (hunk 1 above)
Comment (banker) [reply content/ri/using-redisinsight/troubleshooting.md:32]: I'm more concerned this sentence: "RedisInsight should work fine behind a reverse proxy like NGINX for most use-cases." It's too vague "should work fine...for most use cases". Can we make this clearer?

Comment (bmansheim) [reply content/ri/using-redisinsight/troubleshooting.md:32]: @slemeur Can you provide more detail about using RedisInsight behind a reverse proxy?

**Pull request merged**